source/help/apps/desktop-changelog.md

```diff
@@ -0,0 +1,333 @@
+# Desktop Application Changelog
+
+## Release v3.4.0
+
+Expected release date: End of September 2016
+
+This release contains a security update and it is highly recommended that users upgrade to this version.
```

Comment:
Propose opening a ticket to setup tracking for Desktop App security issues on http://about.mattermost.com/security-updates/

This might mean breaking out the page into sections for server and apps, and we should figure out how to track these internally, e.g. a new column in security issue spreadsheet for component (Server, Desktop App, Mobile App, etc.)